src/client/components/form-elements/DateInput.jsx

```diff
@@ -58,9 +58,18 @@ class DateInput extends React.Component {
           validateIfDirtyDate(day, month, year, isValidDate);
     }
 
+    // Calculate required.
+    let requiredSpan = undefined;
+    if (this.props.required) {
+      requiredSpan = <span className="hca-required-span"> *</span>;
+    }
+
```

Comment:
I think this looks good for now but I wonder if, for future forms, we should have a <requiredAsterisk/>. Maybe for component library down the road.